examples/basicappbzlmod/.bazelrc

```diff
@@ -0,0 +1,9 @@
+common --enable_bzlmod --lockfile_mode=off
+
+# Disabled to avoid //external:android deps in bazel itself
+common --android_sdk=@androidsdk//:sdk
+common --optimizing_dexer=//:optimizing_dexer
```

Comment:
this is a workaround to avoid having this target in bazel reference `//external:android`